HealthMonitoring.SelfHost/HealthMonitoring.SelfHost.csproj

```diff
@@ -15,6 +15,20 @@
     <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\</SolutionDir>
     <NuGetPackageImportStamp>
     </NuGetPackageImportStamp>
+    <Install>true</Install>
```

Comment:
This was automatically (not manually) installed, I am not sure what process; lemme know if i should remove